src/main/resources/application.yml

```diff
@@ -174,3 +174,7 @@ webclient:
     max-retry-attempts: 2
     connect-timeout-seconds: 3
     read-timeout-seconds: 3
+
+pagination:
+  max-size:
+    casenotes: 10
```

Comment:
what's the reason for enforcing this?

Reply:
I added it as a separate commit; if you think it's unnecessary then I can drop the commit.

Currently, for flexibility, the caller can make a pagination request specifying the page number and the page size of the resource they want.

I believe that the interventions-service should be responsible for enforcing a max-size as a protection against pagination causing performance issues.